banzai/munge.py

```diff
@@ -19,18 +19,21 @@ def munge(image):
 
     # 1m SBIGS
     elif '1m0' in image.instrument.type:
-        image.header['SATURATE'] = (46000.0, '[ADU] Saturation level used')
+        if image.header.get('SATURATE', 0.0) == 0.0:
+            image.header['SATURATE'] = (46000.0, '[ADU] Saturation level used')
     elif '0m4' in image.instrument.type or '0m8' in image.instrument.type:
-        image.header['SATURATE'] = (56000.0, '[ADU] Saturation level used')
```

Comment:
For consistency, do the  same treatment for 'MAXLIN' keyword with the identical default value of 64000 if not present.